pages/admin/finance/index.tsx

```diff
@@ -64,29 +100,36 @@ export default function financePage() {
       });
     }
     if (actualCtx !== null) {
+      if (Chart.getChart(actualCtx)) {
+        Chart.getChart(actualCtx)?.destroy();
+      }
       const chart = new Chart(actualCtx, {
         type: 'pie',
         data: {
-          labels: ['Research Funding', 'Dev funding', 'Food', 'Merchandise', 'Other'],
+          labels: divisionNames,
           datasets: [
             {
               label: '% of Budget',
```

Comment:
I believe this should be something like `Amount used` or anything that signify how much they really spent instead of percentage.